WooCommerce/Classes/ViewRelated/Hub Menu/HubMenu.swift

```diff
@@ -62,11 +63,19 @@ struct HubMenu: View {
             NavigationLink(destination: CouponListView(siteID: viewModel.siteID), isActive: $showingCoupons) {
                 EmptyView()
             }.hidden()
+            LazyNavigationLink(destination: viewModel.getReviewDetailDestination(), isActive: $viewModel.showingReviewDetail) {
+                EmptyView()
+            }
         }
         .navigationBarHidden(true)
         .background(Color(.listBackground).edgesIgnoringSafeArea(.all))
     }
 
+    func pushReviewDetailsView(using parcel: ProductReviewFromNoteParcel) {
+        viewModel.setProductReviewFromNoteParcel(parcel)
+        viewModel.showingReviewDetail = true
```

Comment:
nit: is it possible to merge these two calls into one function, so that it's impossible to call one without the other? like:
```suggestion
        viewModel.showReviewDetails(using: parcel)
```

and it sets `showingReviewDetail` to `true` internally in the view model. also, could be an opportunity for unit tests 😄